pkg/cmd/image/remove.go

```diff
@@ -88,18 +89,23 @@ func Remove(ctx context.Context, client *containerd.Client, args []string, optio
 			return nil
 		},
 	}
+
+	var errs []string
 	for _, req := range args {
```

Comment:
It's possible (`nerdctl container rm -f` also has similar logic when `Walk`). We can rename the `forceAll` to `all` (meaning `Walk all and return all joined errors if there is any`), and add a `force` to `WalkAll` for this `if force then log err` logic.

I didn't incorporate this `if force then log err` logic into `WalkAll` yet since it's only used once here, but I can add it if that sounds good.

Reply:
Will something like this ensure that you do not have to change WalkAll?
```
err := walker.WalkAll(...)
if err != nil {
                logrus.WithError(err)
}
```

Reply:
This also works except that all errors will be printed together at the end. Will this mess overall output (or will this be okay)? For example, if the object produces multiple logs + an error at the end, this change will keep the logs in original place but move the error to the end of the command output, together with other errors.

Reply:
it should be fine :) after all we're already doing this when `Force == false` (accumulate err in `errs`)